packages/tailwind-config/index.css

```
@@ -127,6 +164,13 @@ body {
   }
 }
 @utility scrollbar-sm {
+  scrollbar-width: auto;
+  scrollbar-color: auto;
+
+  &:hover {
```

Stylelint (17.14.0)

[error] Missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)